.eslintrc

```diff
@@ -0,0 +1,35 @@
+{
+  "globals" : {
+    "rawData": true,
+    "app": true,
+    "Handlebars": true,
+    "page": true,
+    "$": true,
+    "marked": true,
+    "hljs": true
+  },
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "ecmaFeatures": {"impliedStrict": true}
+  },
+  "env": {
+    "browser": true,
+    "jquery": true,
+    "node": true,
+    "es6": true
+  },
+  "rules": {
+    "eqeqeq": ["error", "always"],
+    "no-template-curly-in-string": "error",
+    "no-console": "off",
+    "no-undefined": "off",
+    "indent": ["error", 2],
+    "quotes": ["warn", "single", {"allowTemplateLiterals": true}],
+    "no-multi-spaces": ["warn", {"exceptions": { "VariableDeclarator": true }}],
+    "no-trailing-spaces": "warn",
+    "new-cap": "warn",
+    "no-redeclare": ["error", { "builtinGlobals": true }],
+    "no-unused-vars": "off"
```

Comment:
You should add a semi-colon rule to this linter:
```
"semicolon": [true, "always"]
```

Reply:
I added it but now my Atom gives me constant error messages. Is it required?

Reply:
Sorry, I got that from the docs for a linter, but apparently it was incorrect. The proper syntax for the semi-colon rule is:
```
"semi": ["error", "always"]
```